components/authz-service/server/v1/server_test.go

```diff
@@ -85,8 +91,23 @@ func setup(t *testing.T) map[string]authz.AuthorizationClient {
 		t.Fatalf("couldn't initialize pg config for tests: %s", err.Error())
 	}
 
+	dataMigrationConfig, err := migrationConfigIfPGTestsToBeRun(l, "../../storage/postgres/datamigration/sql")
```

Comment:
We've added the datamigration config to server/postgres initialization, so we've had to add it in v1 to maintain the interface/keep things compiling